core/domain/translatable_object_registry_test.py

```diff
@@ -24,44 +24,59 @@
 class TranslatableObjectRegistryUnitTests(test_utils.GenericTestBase):
     """Test the Registry class in translatable_object_registry."""
 
-    def test_get_object_class_method(self):
+    def test_get_object_class_method(self) -> None:
         """Tests the normal behavior of get_object_class()."""
         retrieved_class = (
             translatable_object_registry.Registry.get_object_class(
                 'TranslatableHtml'))
         self.assertEqual(retrieved_class.__name__, 'TranslatableHtml')
 
-    def test_nontranslatable_class_is_not_gettable(self):
+    def test_nontranslatable_class_is_not_gettable(self) -> None:
         """Tests that trying to retrieve a non-translatable class raises an
         error.
         """
-        with self.assertRaisesRegex(
-            TypeError, 'not a valid translatable object class'):
-            translatable_object_registry.Registry.get_object_class(
+        with self.assertRaisesRegex( # type: ignore[no-untyped-call]
+            TypeError, 'not a valid translatable object class'
+        ):
+            # TODO(#13059): After we fully type the codebase we plan to get
```

Comment:
The linked issue is already closed, either reopen it with a reason or create a new one.

Reply:
@vojtechjelinek i have reopend the issue with an appropriate comment. PTAL